app/services/atmosphere/appliance_vms_manager.rb

```diff
@@ -91,11 +91,20 @@ def instantiate_vm(tmpl, flavor, name)
     end
 
     def start_vm_on_cloud(tmpl, flavor, name)
+      if defined? Air.config.ostnic
+        nic = Air.config.ostnic.nic
+        Rails.logger.info("Using custom NIC: #{nic}")
+      else
+        Rails.logger.info("Using default NIC.")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.